tests/api/test_api.py

```diff
@@ -73,6 +73,13 @@ async def test_get_search_content_type(app_client):
     assert resp.headers["content-type"] == "application/geo+json"
 
 
+async def test_landing_links(app_client):
+    """test landing page links."""
+    landing = await app_client.get("/")
+    assert landing.status_code == 200, landing.text
+    assert "Queryables" in [link.get("title") for link in landing.json()["links"]]
```

Comment:
make sure the `Queryables` links is added (from stac-fastapi core client)

Reply:
I don't understand the comment in context of your comment here: https://github.com/stac-utils/stac-fastapi/pull/736#issuecomment-2262873514

Reply:
Meant that the link for the queryables endpoint is added in the landing page response directly from the base client definition in stac-FastAPI so this test is kinda re-testing something that should be tested in stac-FastAPI repo